### PR TITLE
[FIX] web: forward "debug" settings when log in

### PR DESCRIPTION
**Pull request by rim-odoo**

### CODE REVIEW

**Comment by rim-odoo** [comment]
@antonylesuisse can you review this?

**Comment by antonylesuisse** [comment]
not very nice, maybe an odoo.http.redirect helper with debug support? @KangOl ?

**Comment by KangOl** [comment]
You're right. Why not fix (and use) `local_redirect` ?

File: addons/web/views/webclient_templates.xml [hunk 1]
```diff
@@ -227,7 +227,7 @@
 
     <template id="web.login" name="Login">
         <t t-call="web.login_layout">
-            <form class="oe_login_form" role="form" t-attf-action="/web/login{{ '?debug' if debug else '' }}" method="post" onsubmit="this.action = this.action + location.hash">
+            <form class="oe_login_form" role="form" t-attf-action="/web/login{{ '?debug=assets' if debug == 'assets' else '?debug' if debug else '' }}" method="post" onsubmit="this.action = this.action + location.hash">
```
Comment (KangOl) [inline addons/web/views/webclient_templates.xml]: Be future proof: use `{{ '?debug=%s' % debug if debug else '' }}`

**Comment by KangOl** [comment]
Also this can be a fix for 9.0

**Comment by rim-odoo** [comment]
Fixed in https://github.com/odoo/odoo/commit/dd4980b0a85873962a57836a431af1718676ffc9

**Pull request closed without merging**